gateway/src/task.rs

```diff
@@ -137,15 +137,14 @@ pub fn start() -> impl Task<ProjectContext, Output = Project, Error = Error> {
 
 pub fn start_idle_deploys() -> impl Task<ProjectContext, Output = Project, Error = Error> {
     run(|ctx| async move {
-        match ctx.state.refresh(&ctx.gateway).await {
-            Ok(Project::Ready(mut ready)) => {
-                let api_key = ctx.api_key;
-
-                ready.start_last_deploy(api_key).await;
+        match ctx.state {
+            Project::Ready(mut ready) => {
```

Comment:
Aren't the idled projects in `ProjectStopped` state?

Reply:
They were started first before we'll try to restore the last deploy
https://github.com/shuttle-hq/shuttle/blob/bc38d3644c4bf841b3e74355cf9e534565a416c2/gateway/src/service.rs#L724-L726

Reply:
Ah, right. Thanks!

nit: Can we rename this method to `start_last_running_deployment`?